tell_me_your_secrets/__main__.py

```diff
@@ -127,15 +130,20 @@ def match(self, file_path: str, file_content: str) -> MatchResult:
 
 
 class SignatureRecognizer:
-    def __init__(self, config_object: dict, search_path: str, use_gitignore: bool, print_results=VERBOSE_OUTPUT,
-                 write_results=SAVE_ON_COMPLETE, output_path=DEFAULT_OUTPUT_PATH, user_filters: list = []):
+    def __init__(self, config_object: dict, search_path: str, use_gitignore: bool, processes_count: int,
+                 print_results=VERBOSE_OUTPUT, write_results=SAVE_ON_COMPLETE, output_path=DEFAULT_OUTPUT_PATH,
+                 user_filters: Optional[list] = None):
+        self.start_time = time.time()
         self.search_path = search_path
         self.use_gitignore = use_gitignore
         if use_gitignore:
             gitignore_file = os.path.join(search_path, '.gitignore')
             if os.path.exists(gitignore_file):
                 module_logger.debug(f'Using gitignore file: {gitignore_file}')
                 self.gitignore_matcher = parse_gitignore(gitignore_file)
+            else:
```

Comment:
shouldn't it be in the same level of indent with `if use_gitignore`? asking based on the property you have set based on else statement.

Reply:
The logic could probably be simplified further but the logic which uses this failed when the matcher was not set. This means that it is set to false and thus not used.